### DISCUSSION TITLE
How to create my own ML library?

### DISCUSSION THREAD

**Discussion by namnaku87** [shankarpandala/lazypredict#409] [General]
Hi Shankar,

The Lazy predict ML library is fantastic to work you put into ML library. 

I want to obtain your suggestions and advice on how to start my own library.

**Comment by BradKML** [comment]
Here is a bit of late advice @namnaku87
1. Try to get familiar with Cursor or other open source equivalents of AI coding tools
2. Fork and modify this codebase like this other dude https://github.com/nityansuman/lazypredict-nightly
3. Fix up every major issue mentioned in this repo so that people can have an easier life
